client/src/pages/Discovery/SearchForm/SimpleSearch/SimpleSearch.tsx

```diff
@@ -0,0 +1,63 @@
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
+import SearchIcon from '@material-ui/icons/Search';
+import { Box, Button, CircularProgress, InputBase, TextField } from '@material-ui/core';
+import useStyles from './useStyles';
+interface Props {
+  handleSimpleSubmit: (
+    {
+      search,
+    }: {
+      search: string;
+    },
+    {
+      setSubmitting,
+    }: FormikHelpers<{
+      search: string;
+    }>,
+  ) => void;
+}
+export default function SimpleSearch({ handleSimpleSubmit }: Props): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Formik
+      initialValues={{ search: '' }}
+      validationSchema={Yup.object().shape({
+        search: Yup.string(),
+      })}
+      onSubmit={handleSimpleSubmit}
+    >
+      {({ handleSubmit, handleChange, values, touched, errors, isSubmitting }) => (
+        <form onSubmit={handleSubmit} noValidate>
+          <div className={classes.root}>
+            <div className={classes.search}>
+              <div className={classes.searchIcon}>
```

Comment:
Same as comment above for flexbox using Box component

Reply:
changed to use Box components